packages/blocks/src/image-block/utils.ts

```diff
@@ -67,30 +67,11 @@ async function getImageBlob(model: ImageBlockModel) {
   if (!sourceId) {
     return null;
   }
-
   const doc = model.doc;
   const blob = await doc.blob.get(sourceId);
-
   if (!blob) {
     return null;
   }
-
-  if (!blob.type) {
```

Comment:
I suggest that we not delete file-type, because affine's sync backend does not store `blob.type`. You can expect all `blob.types` to be empty `""`. We can only calculate them through `file-type` when type is needed.

As for the buffer issue of file-type v19 in the browser, affine can workaround by adding `node:buffer` polyfill.